frontend/public/components/app-contents.tsx

```diff
@@ -682,6 +682,15 @@ const AppContents: React.FC<{}> = () => {
 
             {inactivePluginPageRoutes}
 
+            <LazyRoute
+              path="/importyamlresults"
```

Comment:
Since we agreed that we don't need to code to design and its ok to have the project selector here, we can reduce this functionality into a component that resides on the existing import page instead of a new page.